### PR TITLE
Add instance type to lvl 5 for ec2 and linode hosts

### PR DESCRIPTION
**Pull request by shortdudey123**
Adds instance type output for level 5 output.  Supports ec2 and Linode hosts.

Sample output:
```
box01.example.com (us-west-2b, _default, last seen 2015-01-28 16:30:02 -0800, instance type m1.small)
```

### CODE REVIEW

File: lib/lita/handlers/enhance/node.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@ def render(level)
           when 2 then "#{name} (#{dc})"
           when 3 then "#{name} (#{dc}, #{environment})"
           when 4 then "#{name} (#{dc}, #{environment}, last seen #{last_seen_at})"
-          when 5 then "#{fqdn} (#{dc}, #{environment}, last seen #{last_seen_at})"
+          when 5 then "#{fqdn} (#{dc}, #{environment}, last seen #{last_seen_at}, instance type #{instance_type})"
```
Comment (dougbarth) [inline lib/lita/handlers/enhance/node.rb]: Let's get rid of the ```instance type``` label. I think the value is pretty self explanatory.

**Comment by shortdudey123** [comment]
@dougbarth any updates merging this?

**Comment by dougbarth** [comment]
Hey @shortdudey123,

Could you share what your motivation is for this PR? At least for me personally, I don't know that adding instance sizing to text that has opaque identifiers adds a ton of value. My intention with enhance isn't to expose every bit of machine metadata, but instead to make it easy to fill in commonly useful details when looking at data like log lines, netstat output, etc.

**Comment by shortdudey123** [comment]
If i remember right, you asked me for it a while back :p

If its not needed anymore, go ahead and close

**Comment by dougbarth** [comment]
Hmm, okay. I don't think we need it anymore, so I will close this.

Sorry for the long neglected PR. :person_frowning:

**Comment by shortdudey123** [comment]
:+1: np

**Pull request closed without merging**